dom-querying/index.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html>
+<head>
+  <title>Example Domain</title>
+  <meta charset="utf-8">
+  <meta http-equiv="Content-type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {
+      background-color: #f0f0f2;
+      margin: 0;
+      padding: 0;
+      font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+    }
+
+    div {
+      width: 600px;
+      margin: 5em auto;
+      padding: 2em;
+      background-color: #fdfdff;
+      border-radius: 0.5em;
+      box-shadow: 2px 3px 7px 2px rgba(0, 0, 0, 0.02);
+    }
+
+    a:link,
+    a:visited {
+      color: #38488f;
+      text-decoration: none;
+    }
+
+    @media (max-width: 700px) {
+      div {
+        margin: 0 auto;
+        width: auto;
+      }
+    }
+  </style>
+</head>
+<body>
+  <div>
+    <h1>Example Domain</h1>
+    <p id="explanation">
+      This domain is for use in illustrative examples in documents. You may use this
+      domain in literature without prior coordination or asking for permission.
+    </p>
+    <p class="hint">
+      <a class="example-link mdn" href="https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors">
+        Locating DOM elements using selectors
+      </a>
+    </p>
+    <p>
+      <a class="example-link" href="https://www.iana.org/domains/example">
+        More information...
+      </a>
+    </p>
+  </div>
+  <script>
+    document.querySelector('h1');
+    console.log(document.querySelector('h1'));
+    console.dir(document.querySelector('h1'));
+
+    document.querySelector('#explanation');
+    console.log(document.querySelector('#explanation'));
+    console.dir(document.querySelector('#explanation'));
+
+    document.querySelector('.hint');
+    console.log(document.querySelector('.hint'));
+    console.dir(document.querySelector('.hint'));
+
+    document.querySelectorAll('p');
+    console.log(document.querySelectorAll('p'));
+
+    document.querySelectorAll('.example-link');
+    console.log(document.querySelectorAll('.example-link'));
```

Comment:
This javascript code should be in your `main.js` file. Here, you can use the `src` attribute to link the `main.js` file similar to how you would use a `link` tag in the head for your CSS stylesheet.